packages/extension/src/view/devtools/components/infoCard/learnMoreDropdown.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies.
+ */
+import React, { useState } from 'react';
+
+/**
+ * Internal dependencies.
+ */
+import PSInfo from './PSInfo.json';
```

Comment:
I think rather than importing the data required to display we should fetch it like we are fetching the [cookieDB](https://github.com/GoogleChromeLabs/ps-analysis-tool/blob/develop/packages/extension/src/utils/fetchCookieDictionary.ts). In the future, any static data can then be fetched from a server with just a url change and no changes to the presentational components.

@mayan-000 @sayedtaqui opinions?

Reply:
Yup, I think the same method can be used here too, making it convenient to fetch data at a later stage if a URL change happens.